releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,19 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.2.3 - April 16, 2025
+
+### Bug Fixes
+
+* Fixed a bug with incorrect `toYaml` usage in service account templates. [#106](https://github.com/Azure/AppConfiguration-KubernetesProvider/issues/106)

```

Comment:
We should describe the symptoms of the bug, so users know whether it's relevant to them.